airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/manifest_component_transformer.py

```diff
@@ -74,6 +74,8 @@
     "SimpleRetriever.partition_router": "CustomPartitionRouter",
 }
 
+_PROPAGATION_EXCLUSION_TYPES = {"InlineSchemaLoader"}  # propagation of extra parameters leads to invalid JSON schemas
```

Comment:
I was wondering if we should use `InlineSchemaLoader.schema()["properties"]["type"]["enum"][0]` to make sure that if the enum changes, the dependency to this is explicit and this is updated as well

Reply:
This would mean a wider range of impact though. For example, `spec.connection_specification` would be affected by this. I don't see any parameters for this as well and it seems fine to say that there wouldn't ever be for the same reason we want to exclude `InlineSchemaLoader.schema` though so I would be fine with this